### PR TITLE
Webapp blob connection

### PR DESCRIPTION
**Pull request by JeffBow**
### Added New Template
Added a new WebApp deployment template that uses a Custom connection string to access Azure Blob Storage using Microsoft.Web/sites/config

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JeffBow__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@JeffBow__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: webapp-blob-connection/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Create a web app on Azure with Blob Storage connection string<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fwebapp-with-blobconnection%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
```
Comment (singhkay) [inline webapp-blob-connection/README.md]: Deploy button code is not right, it's not rendering

File: webapp-blob-connection/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,166 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "WebAppSvcPlanName": {
```
Comment (singhkay) [inline webapp-blob-connection/azuredeploy.json]: parameters need to be camelCase

File: webapp-blob-connection/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Create a web app on Azure with Blob Storage connection string<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fwebapp-with-blobconnection%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline webapp-blob-connection/README.md]: Please rename the folder to 201-webapp-blob-connection

File: webapp-blob-connection/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        "description": "Name of web site."
+      }
+    },
+    "StorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage Account to access blob storage."
+      }
+    },
+    "StorageAccountKey": {
```
Comment (singhkay) [inline webapp-blob-connection/azuredeploy.json]: This doesn't need to be a parameter. Use the listKeys template function

**Comment by singhkay** [comment]
merged. CI won't deploy since this requires presetup

**Pull request merged**